### ISSUE TITLE
The last update killed the proxy Authentication header

### ISSUE THREAD

**Issue by MartiONE** [berstend/puppeteer-extra#89]
After the last update today the `Proxy-Authorization` header that will authentificate the browser while using  `--proxy-server=${proxyUrl}` stopped working, the website returns a `407 Response`.

Proxy used is: https://support.scrapinghub.com/support/solutions/articles/22000220800-using-crawlera-with-puppeteer

**Comment by berstend** [comment]
This is probably connected to #86 so it's `accept-language` plugin related.

In that (new) evasion we hook into requests to modify the request header correctly, auth and apparently proxy auth are handled super weirdly in chromium so I assume that's the issue here.

I'll look into it, for the time being you can just disable that specific stealth plugin and all should work:

```js
pluginStealth.enabledEvasions.delete('accept-language')
```

**Comment by berstend** [comment]
@MartiONE as per the [documentation](https://support.scrapinghub.com/support/solutions/articles/22000220800-using-crawlera-with-puppeteer), are you using `page.setExtraHTTPHeaders` to set the `Proxy-Authorization` header?

**Comment by berstend** [comment]
@MartiONE are you sure it's the stealth plugin that's causing your issues?

```js
const puppeteer = require("puppeteer-extra")

const StealthPlugin = require("puppeteer-extra-plugin-stealth")
const stealth = StealthPlugin()
puppeteer.use(stealth)

puppeteer
  .launch({
    headless: false,
    ignoreHTTPSErrors: true,
    args: ["--proxy-server=proxy.crawlera.com:8010"]
  })
  .then(async browser => {
    const page = await browser.newPage()
    await page.goto("http://httpbin.org/headers")

    const content = await page.content()
    console.log(content)
    await browser.close()
  })
```

This triggers the auth popup with and without using the stealth plugin.

**Comment by berstend** [comment]
Also:

https://blog.apify.com/how-to-make-headless-chrome-and-puppeteer-use-a-proxy-server-with-authentication-249a21a79212
https://www.npmjs.com/package/proxy-chain

**Comment by berstend** [comment]
Closing as there was no response from @MartiONE.

**Comment by MartiONE** [comment]
Sorry @berstend I did not get any notification on this specific issue, no idea why.

The problem only appeared on the new version at that time of the `puppeteer-extra-plugin-stealth`, I rolled it back to  a previous version (2.2.2) and it worked without any problem. Also tried to update it as right now and it also worked perfectly, no idea what was changed.

I am using page.setExtraHTTPHeaders to set the Proxy-Authorization header.

Consider this issue solved, thanks for your time.
